portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsSubmissions.testcase

```diff
@@ -499,6 +499,112 @@ definition {
 		}
 	}
 
+	@description = "Verify if is not possible navigate through pages on public pages with not contains validation error"
+	@priority = 5
+	test CannotNavigateThroughThePagesOnPublicPagesWithDoesNotContainsValidationError {
+		property portal.acceptance = "true";
+		property portal.upstream = "true";
```

Comment:
Remove this property: `property portal.upstream = "true";`

Acceptance property replaces the behavior for it.